### DISCUSSION TITLE
Twitter changed address to "x.com" and I can't access Dark Reader settings anymore

### DISCUSSION THREAD

**Discussion by SixelAlexiS90** [darkreader/darkreader#12736] [Q&A] [answered]
As the title say I can't access twitter.com since it instantly redirect to x.com and I need to copy the Dark Reader settings from twitter.com, how can I do that?
Thank you!

**Comment by SixelAlexiS90** [comment]
Well, my solution was to disable internet to prevent the redirect and so to let Dark Reader access the twitter.com page and manually copy the settings.
It would be great to see the per site settings even without accessing them.

**Comment by CamoCatX** [comment, marked as answer]
You can do this by exporting the your theme (in chromium browsers the button would show here: chrome-extension://eimadpbcbfnmbkopoojfekhnkhdbieeh/ui/options/index.html ) and then go into the file, and find all mentions of "twitter.com". You can usually use the shortcut CTRL+F to find that. Then replace the mentions of twitter.com to x.com, then import the json file back to darkreader.

**Reply by SixelAlexiS90** [reply]
> You can do this by exporting the your theme (in chromium browsers the button would show here: chrome-extension://eimadpbcbfnmbkopoojfekhnkhdbieeh/ui/options/index.html ) and then go into the file, and find all mentions of "twitter.com". You can usually use the shortcut CTRL+F to find that. Then replace the mentions of twitter.com to x.com, then import the json file back to darkreader.

I've actually searched the same info by looking in the .log file in:
\AppData\Local\Google\Chrome\User Data\Default\Local Extension Settings\eimadpbcbfnmbkopoojfekhnkhdbieeh

And thought the results were old/wrong since I saw the one UNDER the site and not the one on top of the site, my bad X'D
Thank you, I've did an .json backup just in case something goes wrong, thank you!
